torch/csrc/deploy/interpreter/freeze.py

```diff
@@ -225,7 +225,9 @@ def compile_file(self, path: Path, top_package_path: Path):
         c_name = "M_" + module_mangled_name
 
         with open(path, "r") as src_file:
-            co = compile(src_file.read(), path, "exec")
+            # instead of passing in the real build time path to 'compile', we pass in a marker instead
```

Comment:
Also: Maybe expand this comment a little bit, explaining why we want to set a path marker and what problem it avoids.